modules/perception/lidar/lib/segmentation/ncut/ncut.cc

```diff
@@ -363,9 +369,8 @@ void NCut::NormalizedCut(float ncuts_threshold, bool use_classifier,
   while (!job_stack.empty()) {
     curr = job_stack.top();
     job_stack.pop();
-    AINFO << "curr size " << curr->size();
 #ifdef DEBUG_NCUT
-    DEBUG << "curr size " << curr->size();
+    AINFO << "curr size " << curr->size();
```

Comment:
Use ADEBUG for debugging purposes.

Reply:
oh, for this, this is protected compilation flag. so in production code, it will not be activated.